### DISCUSSION TITLE
Reports Being Displayed Twice

### DISCUSSION THREAD

**Discussion by 0xNazgul** [solodit/feedback#34] [Bug Report]
Not fully sure if this was brought up anywhere else besides the discussion board. But some reports are being displayed twice with some minor detail changes. For example, a search with:
- The keyword: "farm"
- Impact: high, medium
- Reported after: 01/01/2022

On the first page the issue *"[M-03] reward will be locked in the farm if no LP join the pool at epoch.startBlock"* is seen twice [here](https://solodit.xyz/issues/m-03-reward-will-be-locked-in-the-farm-if-no-lp-join-the-pool-at-epochstartblock-code4rena-jpegd-jpegd-contest-git) and [here](https://solodit.xyz/issues/m-03-reward-will-be-locked-in-the-farm-if-no-lp-join-the-pool-at-epochstartblock-code4rena-jpegd-jpegd-contest-git_). 
 
Another example further down the results in the same search is *"[M-05] Rewards will be locked if user transfer directly to pool without using deposit function"* can be seen twice [here](https://solodit.xyz/issues/m-05-rewards-will-be-locked-if-user-transfer-directly-to-pool-without-using-deposit-function-code4rena-jpegd-jpegd-contest-git) and [here](https://solodit.xyz/issues/m-05-rewards-will-be-locked-if-user-transfer-directly-to-pool-without-using-deposit-function-code4rena-jpegd-jpegd-contest-git_). 

Seems to be only an issue with Code4rena reports as far as I have seen. This makes results noisy and a little annoying when reading.

**Comment by hans-cyfrin** [comment]
We have a few findings that would show up twice (probably because of an unnoticeable change in the finding meta info that results in the finding hash). Because notes and discussions can be created for both, it is not straightforward how to handle this. We would acknowledge this and won't fix for now.
